gui/gui/src/ts/components/search/NodeSearchBox.tsx

```diff
@@ -69,6 +73,11 @@ export const NodeSearchBox: React.FunctionComponent<{
             ) : null
           }
         ></InputBase>
+        <NodeSearchSuggestions
```

Comment:
I'm not sure why you implemented your own Autocomplete, when @material-ui has one and there are others like react-select that work well if @material-ui's doesn't.

Reply:
I'm not totally opposed to implementing bespoke stuff, I'd just like to understand why the existing alternatives were insufficient.

Reply:
Switched over to @material-ui/lab/Autocomplete